grading-assigner.py

```diff
@@ -135,33 +155,45 @@ def request_reviews(token):
                 # Refreshing a request is more costly than just loading
                 # and only needs to be done to ensure the request doesn't
                 # expire (1 hour)
-                logger.info('0-0-0-0-0-0-0-0-0-0- refreshing request 0-0-0-0-0-0-0')
+                logger.info('0-0-0-0-0-0-0-0-0-0-\
+                            refreshing request\
+                            0-0-0-0-0-0-0')
                 current_request = refresh_request(current_request)
             else:
                 logger.info('Checking for new assignments')
                 # If an assignment has been made since status was last checked,
                 # the request record will no longer be 'fulfilled'
-                url = GET_REQUEST_URL_TMPL.format(BASE_URL, current_request['id'])
+                url = GET_REQUEST_URL_TMPL.format(BASE_URL,
+                                                  current_request['id'])
                 get_req_resp = requests.get(url, headers=headers)
-                current_request = get_req_resp.json() if me_req_resp.status_code == 200 else None
+                current_request = get_req_resp.json() if\
+                    me_req_resp.status_code == 200 else None
 
         current_request = alert_for_assignment(current_request, headers)
         if current_request:
-            # Wait 2 minutes before next check to see if the request has been fulfilled
+            # Wait 2 minutes before next check
+            # to see if the request has been fulfilled
             time.sleep(120.0)
 
 if __name__ == "__main__":
-    cmd_parser = argparse.ArgumentParser(description =
-	"Poll the Udacity reviews API to claim projects to review."
-    )
-    cmd_parser.add_argument('--auth-token', '-T', dest='token',
-	metavar='TOKEN', type=str,
-	action='store', default=os.environ.get('UDACITY_AUTH_TOKEN'),
-	help="""
-	    Your Udacity auth token. To obtain, login to review.udacity.com, open the Javascript console, and copy the output of `JSON.parse(localStorage.currentUser).token`.  This can also be stored in the environment variable UDACITY_AUTH_TOKEN.
-	"""
-    )
-    cmd_parser.add_argument('--debug', '-d', action='store_true', help='Turn on debug statements.')
+    cmd_parser = argparse.ArgumentParser(description="Poll the Udacity reviews\
```

Comment:
Oof. Does PEP8 really recommend aligning arguments like that? I've always thought that's ugly since it leads to messy whitespace diffs if the first line ever changes. But I don't write Python often so if that's the standard I'm fine with it. 👍